### PR TITLE
feat: new option for trailing comma

### LINKED ISSUE #798: [feature] support difference value for trailing commas
Values:
- `none` doesn't print any trailing commas.
- `php5.0` print trailing commas for `array`
- `php7.2` include all for `php7.2` + `list` and `use` (grouped namepaces only).
- `php7.3` include all for `php7.2` + `function calls` (https://wiki.php.net/rfc/trailing-comma-function-calls).
- `all` is latest stable `PHP` release (right now it is `php7.2`, in near future `php7.3`).

Somebody know when `PHP` starts support trailing commas for `array` ? :smile: But `php5.0` looks good for me because minimum supported version is `5`.

Feel free to feedback

### PR DESCRIPTION
**Pull request by evilebottnawi**
fixes #798

### CODE REVIEW

File: src/printer.js [hunk 1]
```diff
@@ -58,10 +58,24 @@ const {
...
+    // fallthrough
+    case "php7.2":
+      if (level === "php7.2") {
+        return true;
+      }
+    // fallthrough
+    case "php5":
+      if (level === "php5") {
+        return true;
+      }
```
[the remark this answers is not in the document]
Comment (evilebottnawi) [reply src/printer.js]: @czosel yep, i can rebase after merge https://github.com/prettier/plugin-php/pull/899 and you can see snapshot for option `7.2`

[the remark this answers is not in the document]
Comment (evilebottnawi) [reply src/printer.js]: /cc @czosel 
You can find example here https://github.com/prettier/plugin-php/blob/15ddd1ee4b35f016f6e7373b8ffdf375c7b6f565/tests/trailing_commas/__snapshots__/jsfmt.spec.js.snap#L868, https://github.com/prettier/plugin-php/blob/15ddd1ee4b35f016f6e7373b8ffdf375c7b6f565/tests/trailing_commas/__snapshots__/jsfmt.spec.js.snap#L1939 and https://github.com/prettier/plugin-php/blob/15ddd1ee4b35f016f6e7373b8ffdf375c7b6f565/tests/trailing_commas/__snapshots__/jsfmt.spec.js.snap#L2238

**Comment by evilebottnawi** [comment]
Something wrong with Appveyor, but i think it is problem on their side

File: src/printer.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (czosel) [reply src/printer.js]: :100:

**Pull request merged**